scripts/tag-custom-build.sh

```diff
@@ -40,10 +40,20 @@
 
 set -euo pipefail
 
+use_jj=false
+if [[ "${1-}" == "--jj" ]] ; then
```

Comment:
Nit: it seems that I have to do `scripts/tag-custom-build.sh --jj $SHA`, and `scripts/tag-custom-build.sh $SHA --jj` will silently have the undesired behavior of using `git`. Maybe this is OK (to my knowledge you're the only one using `--jj` :slightly_smiling_face:) but it's worth keeping in mind.

Reply:
Good point. I switched to `getopts` to address the issue..